examples/kubeval/README.md

```diff
@@ -0,0 +1,32 @@
+# Kubeval
+
+The `kubeval` KRM config function validates configuration using kubeval. This
+example invokes the kubeval function using declarative configuration.
+
+## Function invocation
+
+Get this example and try it out by running the following commands:
+
+```sh
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/kubeval .
```

Comment:
we discussed this should also show how to hit a live cluster

Reply:
Kubeval doesn't hit live clusters. It accepts args for --kubernetes-version and for --schema-location and uses a k8s version's schema or the schema passed in to validate resources against. I use --kubernetes-version 1.18.0 in this blueprint.

Reply:
Nice